src/views/SetKeys.js

```diff
@@ -299,34 +335,22 @@ class SetKeys extends React.Component {
       ? props.pointCache[state.point]
       : (() => { throw BRIDGE_ERROR.MISSING_POINT })()
 
-    const isManagementMnemonic =
-      this.state.isManagementSeed &&
-      props.walletType === WALLET_NAMES.MNEMONIC
-
-    const isMasterTicket =
-      props.walletType === WALLET_NAMES.TICKET ||
-      props.walletType === WALLET_NAMES.SHARD
-
-    const networkSeedDisabled = isManagementMnemonic || isMasterTicket;
-
     return (
       <Row>
         <Col>
           <H1>
             { 'Set Network Keys For ' } <code>{ `${ob.patp(state.point)}` }</code>
           </H1>
 
-          <P>
+          <P className="mt-10">
           {
-            `Please enter a network seed for generating and setting your public
-             network authentication and encryption keys.  Your network seed
-             must be a string of 64 characters (containing 0-9, A-Z, a-z).`
+            `Set new authentication and encryption keys for your Arvo ship.`
           }
           </P>
+
           <P>
           {
-             `If you've authenticated with a master ticket or management proxy
-              mnemonic, a seed will be generated for you automatically.`
+            `Once the transaction is sent, please generate your Arvo keyfile immediately.`
```

Comment:
This sentence needs more gravity. We should probably put it in one of those orange warning boxes, like the one we present when you try to configure keys for the very first time.

Might also want to add a brief description why. "Your network seed is not deterministic. Once the transaction ..."